### ISSUE TITLE
Vehicle License Number Plate Detection using YOLOv7 model

### ISSUE THREAD

**Issue by saleena-18** [Niketkumardheeryan/ML-CaPsule#633]
Problem: Traditational CNN models are much slower than YOLOv7 due their complex multi-stage pipeline 

Solution: YOLOv7 is 509% faster and has 2% higher accuracy than Mask-R CNN Model
It requires several times cheaper hardware than other neural networks and can be trained much faster on small datasets without any pre-trained weights.

Approach: We will perform Transfer Learning and fine-tune our CNN (convolutional neural network)/ deep learning model using the yolov7 architecture. Then after performing OCR text detection and extraction, the model will be able to predict vehicle number by taking video and image input.

Further, we can make this take real time video input since YOLO models because of their high speed can be easily used for real-time object detection which can be implemented in practical scenarios such as traffic surveillance systems  in case of non-adherence to traffic rules on roads and highways., whereas CNN models would lag in such a case.

**Comment by saleena-18** [comment]
Please sir,
I'd love to work on this issue since I have prior knowledge of working on YOLO models with Face Detection algorithms, can you please label this issue for 'gssoc' and the 'level.'
This would be practically helpful for the Government too for traffic surveillance in case of non-adherence to traffic rules on roads and highways.
I'd be really grateful if you would assign me this issue. I promise to work diligently on the project.

**Comment by invigorzz313** [comment]
The issue has been assigned.
Do follow the project guidelines for PR.

**Comment by saleena-18** [comment]
@invigorzz313 Sure ma'am! Thankyou so much!
Additionally, could you please add the 'level' label to the issue as well..?

**Comment by invigorzz313** [comment]
Level labels are put on the PRs after reviewing it.

**Comment by sanskritiagr** [comment]
I want to contribute to this. Please assign it to me.
